paragraphs.type_clone.inc

```diff
@@ -213,9 +213,11 @@ function paragraphs_admin_bundle_clone_create_form_submit(array $form, array &$f
         )
       );
       // Redirect to the paragraph types list.
-      backdrop_goto('/admin/structure/paragraphs');
+      $form_state['redirect'] = '/admin/structure/paragraphs';
```

Comment:
@yorkshire-pudding 
Would this be better without the leading `/` in the URL? Sorry I just saw that and it opened the thought that it might be more universal without it, as a fully internal path (e.g. I think potentially the `/` could interfere if Backdrop is installed in a subdirectory or without clean URLs enabled). @argiepiano am I right on this?

Reply:
Will do - that is consistent with all other URLs in the paragraphs module

Reply:
Agreed!